metricbeat/Dockerfile

```diff
@@ -7,7 +7,9 @@ RUN set -x && \
          netcat python-pip virtualenv && \
     apt-get clean
 
+RUN pip install --upgrade pip
 RUN pip install --upgrade setuptools
+RUN pip install --upgrade docker-compose==1.21.0
```

Comment:
I think we should update this also in our docs in a follow up PR: https://www.elastic.co/guide/en/beats/devguide/current/beats-contributing.html#running-testsuite